CHANGES/8250.feature

```diff
@@ -0,0 +1,6 @@
+Added the ALLOW_UNSAFE_ADVISORY_CONFLICT_RESOLUTION configuration option.
```

Comment:
If it would be up to me i would call it  `ALLOW_AUTOMATIC_ADVISORY_CONFLICT_RESOLUTION`

Whether this would be done automatically of left on the user's shoulders to manually resolve - both of the way do not guarantee 100% success ..i think the manual approach would have even smaller success rate realistically speaking.

Reply:
Even without this we do a fair amount of 'automatic' advisory-conflict-resolution.

The name is easy to change, and I can def see AUTOMATIC being a better choice than UNSAFE. OTOH - letting this happen is kinda dangerous, maybe the scarier word is better?

 @goosemania  , @dralley thoughts here?

Reply:
UNSAFE_AUTOMATIC....? :1st_place_medal: 

I agree that we perform automatic conflict resolution already, but my main worry is that automatic doesn't imply that can be an unexpected outcome for the user. UNSAFE or not, I'd keep some scary word in the name.

Reply:
+1 to scary words